src/test/static/class/class-methods.js

```diff
@@ -46,4 +46,14 @@ class Class {
   customInstanceFunctionWithParamsAndPrivateJSDoc() {
     return 8;
   }
+
```

Comment:
In this file, we should probably be using `number` rather than the weird `Number` type.